Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -33,8 +33,14 @@ set(BABYLON_NATIVE_PLAYGROUND_DIR "${CMAKE_CURRENT_LIST_DIR}/../..")
 add_subdirectory(${BABYLON_NATIVE_PLAYGROUND_DIR}/../../ ${BABYLON_NATIVE_PLAYGROUND_DIR}/../../build/Android_${CMAKE_ANDROID_ARCH_ABI}/)
 
 add_library(BabylonNativeJNI SHARED
+    ${BABYLON_NATIVE_PLAYGROUND_DIR}/Shared/Playground/GraphicsApiInfo.cpp
```

Comment:
I've exposed this new `GraphicsApiInfo` class on all platforms for completeness, but maybe we only want to expose it on the platforms running validation tests in the CI, i.e. Win32 only?

Reply:
I wonder if we should restrict the information to just GraphicsApiInfo, since we are already exposing a new API to Javascript couldn't we call it something like "NativeApiInfo" or something and use it to expose more native information in the future?

Reply:
Why not put in TestUtils?